prioritization_fees/src/rpc_data.rs

```diff
@@ -23,6 +23,29 @@ pub struct PrioFeesStats {
     pub cu_consumed: TxAggregateStats,
 }
 
+impl PrioFeesStats {
+    pub fn get_percentile(&self, percentile: f32) -> Option<(u64, u64)> {
+        let index_tx = self
+            .by_tx_percentiles
+            .iter()
+            .enumerate()
+            .find(|(_, v)| **v == percentile)
+            .map(|(ix, _)| ix);
+        let index_cu = self
+            .by_cu_percentiles
+            .iter()
+            .enumerate()
+            .find(|(_, v)| **v == percentile)
+            .map(|(ix, _)| ix);
+        if let Some(index_tx) = index_tx {
```

Comment:
`if let (Some(index_tx), Some(index_cu)) = (index_tx, index_cu) ...` should work and looks a bit cleaner